server/fleet/cron_schedules.go

```diff
@@ -0,0 +1,141 @@
+package fleet
+
+import (
+	"fmt"
+	"net/http"
+	"time"
+)
+
+type CronSchedulesService interface {
```

Comment:
This interface is not used anywhere right?

Reply:
It's used in the Service struct in `server/service/service` that gets created in NewService